script.js

```diff
@@ -0,0 +1,68 @@
+const contactForm = document.querySelector('.book_form');
+const title = document.querySelector('.book_title');
+const author = document.querySelector('.book_author');
+const bookList = document.querySelector('.books-list');
+let titleArray = [];
+let authorArray = [];
+
+function intitializeDocument() {
+  bookList.innerHTML = '';
+  for (let i = (titleArray.length - 1); i > -1; i -= 1) {
+    bookList.innerHTML = `${bookList.innerHTML}<p>${titleArray[i]}</p>
+                            <p>${authorArray[i]}</p>
+                            <button class="remove_button_${i}">remove</button><hr/>`;
+  }
+}
+
+function removeBook(index) {
+  document.querySelector(`.remove_button_${index}`).addEventListener('click', (event) => {
+    const tempArrayTitle = [];
+    const tempArrayauthor = [];
+    for (let j = 0; j < titleArray.length; j += 1) {
+      if (j !== event.currentTarget.index) {
+        tempArrayTitle.push(titleArray[j]);
+        tempArrayauthor.push(authorArray[j]);
+      }
+    }
+    titleArray = tempArrayTitle;
+    authorArray = tempArrayauthor;
+    localStorage.setItem('title', JSON.stringify(titleArray));
+    localStorage.setItem('author', JSON.stringify(authorArray));
+    intitializeDocument();
+    /* eslint-disable */
+    intitializeRemoveButtonEvents();
+   /* eslint-enable */
+  });
+  document.querySelector(`.remove_button_${index}`).index = index;
+}
+
+function intitializeRemoveButtonEvents() {
+  for (let i = (titleArray.length - 1); i > -1; i -= 1) {
+    removeBook(i);
+  }
+}
+
+function addBook() {
+  titleArray.push(title.value);
+  authorArray.push(author.value);
+  localStorage.setItem('title', JSON.stringify(titleArray));
+  localStorage.setItem('author', JSON.stringify(authorArray));
```

Comment:
- [ ]  Currently Title and Author of the same book are being saved in separate array. I strongly suggest you to save Title and Author of 1 book as 1 object in local storage, eg:

```
// Arr of objects
let books = [
  { title: '`title`', author: 'author' },
  { title: '`title`', author: 'author'  },
];

// save to local storage
  const newBooks = JSON.stringify(books);
  localStorage.setItem('books', newBooks); 

// get user data and push new obj into books arr
    const title = form.title.value;
    const author = form.author.value;
    const book = { title, author };
    books.push(book);
```
> This would be a correct approach, because later if you have to add more data lets say Publication date, Publisher etc. You will have to add more arrays and your application will get difficult to manage

Reply:
Thanks for pointing this out. i have made changes to this. Thanks a lot 👍